packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -211,47 +208,6 @@ export class ProjectActions {
     return this.api.launchProject(browser, activeSpec ?? emptySpec, options)
   }
 
-  async launchProjectWithoutElectron () {
-    if (!this.ctx.currentProject) {
-      throw Error('Cannot launch project without an active project')
-    }
-
-    const preferences = await this.api.getProjectPreferencesFromCache()
-    const { browserPath, testingType } = preferences[this.ctx.lifecycleManager.projectTitle] ?? {}
-
-    if (!browserPath || !testingType) {
-      throw Error('Cannot launch project without stored browserPath or testingType')
-    }
```

Comment:
We can re-add this code (currently unused) when we get the actual workflow for this lifecycle down, a lot of this has changed quite a bit relative to the new workflow and probably should be added with e2e tests.